src/css/global/common.css

```diff
@@ -18,6 +18,14 @@ body {
     min-height: 100vh;
 }
 
+a:not([class]),
+a[class=""] {
+    text-decoration: none;
+    color: var(--color-text);
+    border-bottom: 1px solid CurrentColor;
```

Comment:
В HTML и CSS это всё нечувствительно к регистру, поэтому это работает. И всё же это значение пишется в camel case, а не pascal case

Reply:
Хотя, возможно, более правильно писать `currentcolor`. Вот что нагуглил:

https://github.com/ryboe/CSS3/issues/111 - тут чел приводит ссылки на спецификации CSS3, где `currentColor`, и более новую `CSS4`, где `currentcolor`

https://github.com/stylelint/stylelint/issues/4884 - вот тут ещё чел приводит часть цитаты из спецификации, где говорится, что это значение изначально было значением атрибута в svg, а в CSS пришло позже. Поэтому оно единственное, где слова не разделяются дефисом

Вот полная цитата:

> Note: Multi-word keywords in CSS usually separate their component words with hyphens. currentcolor doesn’t, because (deep breath) it was originally introduced in SVG as a property value, "current-color" with the usual CSS spelling. It (along with all other properties and their values) then became presentation attributes and attribute values, as well as properties, to make generation with XSLT easier. Then all of the presentation attributes were changed from hyphenated to camelCase, because the DOM had an issue with hyphen meaning "minus". But then, they didn’t follow CSS conventions anymore so all the properties and property values that were already part of CSS were changed back to hyphenated! currentcolor was not a part of CSS at that time, so remained camelCased. Only later did CSS pick it up, at which point the capitalization stopped mattering, as CSS keywords are ASCII case-insensitive.

Reply:
Но шторм его сейчас в camel case предлагает, давай пока так оставим. В будущем скорее всего это изменится

<img width="606" alt="Screenshot 2021-08-18 at 10 33 54" src="https://user-images.githubusercontent.com/19751269/129856979-41ee424d-5fd3-4097-adea-6b045b078413.png">

Reply:
Если я пишу `currentcolor` то строка не работает. Шторм выдаёт ошибку и в хроме не работает свойство.  
<img width="299" alt="image" src="https://user-images.githubusercontent.com/63367508/129861771-d56d81fc-2df2-431a-b99f-3438623bda94.png">
в случае` currentColor `все ок. 
<img width="294" alt="image" src="https://user-images.githubusercontent.com/63367508/129861932-2afed2d6-a3c3-495d-bd6c-b045860edf83.png">

Reply:
Странно, у меня работает даже заглавными буквами. А `c` точно английская?

Reply:
Ну так или иначе, даже хром при автозаполнении пишет camel case, так что пока стоит так писать